### DISCUSSION TITLE
Button to temporary hide read posts

### DISCUSSION THREAD

**Discussion by Maxplosion** [Docile-Alligator/Infinity-For-Reddit#859] [Duplicate Ideas] [answered]
Feature: a button e.g. in place of "+" button while scrolling on the feed that temporary hides all read posts?

on full refresh e.g. by pulling down to refresh, those posts would be shown again.

this allows to quickly see only new unread posts, while still being able to also see read posts with one pull down refresh.

reason: sometimes you might want to revisit a post that got a lot of comments since the last time you read it.

**Comment by Wladefant** [comment, marked as answer]
Duplicate of #974. Please add your idea there

**Reply by Maxplosion** [reply]
I just found that this has been added by longpressing the '+' button at the bottom right of the page. Exactly what I wanted, thx!

**Reply by Wladefant** [reply]
No problem
